mbf_abstract_nav/src/abstract_execution_base.cpp

```diff
@@ -78,12 +73,13 @@ void AbstractExecutionBase::stop()
   thread_.interrupt();
 }
 
-void AbstractExecutionBase::join(){
+void AbstractExecutionBase::join()
```

Comment:
otherwise clang complains,,, which is weird, cause this is well away from the changes  :thinking: